source/sass/blocks/form-options.scss

```
@@ -30,8 +30,9 @@
     font-size: 36px;
     line-height: 36px;
     padding-bottom: 56px;
-    padding-left: 40px;
     padding-right: 41px;
+    padding-left: 39px;
+    margin-left: -39px;
```

Чтобы легенда в филдсете вела себя одинаково в Хроме и ФФ, надо применить такую вот хитрость

```suggestion
    margin-left: -39px;
    float: left;
```